src/system/data-export.md

```diff
@@ -4,7 +4,7 @@ title: Export
 
 The best way to become familiar with the structure of your database is to export the data and open it in a spreadsheet. Once you become familiar with the process, you’ll find that it is an efficient way to manage large amounts of information.
 
-Special characters—such as the equal sign, greater and less than symbols, single and double quotes, backslash, pipe, and ampersand symbols—can cause problems during data transfer. To ensure that such special characters are correctly interpreted, they can be marked as an “escape sequence.” For example, if the data includes a string of text such as code=”str”, code=”str2”, choosing to enclose the text in double quotes ensures that the original double quotes are understood to be part of the data. When the system encounters a double set of double quotes, it understands that the outer set of double quotes is enclosing the actual data.
+Special characters—such as the equal sign, greater and less than symbols, single and double quotes, backslash, pipe, and ampersand symbols—can cause problems during data transfer. To ensure that such special characters are correctly interpreted, they can be marked as an `escape sequence.` For example, if the data includes a string of text such as code=`str`, code=`str2`, choosing to enclose the text in double quotes ensures that the original double quotes are understood to be part of the data. When the system encounters a double set of double quotes, it understands that the outer set of double quotes is enclosing the actual data.
```

Comment:
I don't think this is a proper change. It is saying: If you need to the actual string to be `code="str"`, then wrap it in double quotes: `"code="str""`. Note that those backticks are markdown ticks to display the string as code.